install/upgrade/versions/1.8.0.sh

```diff
@@ -28,3 +28,7 @@ if [ "$IMAP_SYSTEM" = "dovecot" ]; then
 		sed -i "s/^service auth {/service auth {\n  extra_groups = mail\n/g" /etc/dovecot/conf.d/10-master.conf
 	fi
 fi
+
+if [[ -z "$(command -v jq)" ]]; then
+    apt-get -y install jq
```

Comment:
apt get install can't be set via upgrade version. It is also include as an dependency and should be already available of each install 

https://github.com/hestiacp/hestiacp/blob/7f966da91fb83e9ab1e694d0b7db33bc4002fbe3/src/deb/hestia/control#L9